gtsam/nonlinear/internal/ExecutionTrace.h

```diff
@@ -37,7 +38,13 @@ template<int T> struct CallRecord;
 /// Storage type for the execution trace.
 /// It enforces the proper alignment in a portable way.
 /// Provide a traceSize() sized array of this type to traceExecution as traceStorage.
-static const unsigned TraceAlignment = 16; // 16 bytes is the default alignment used by Eigen.
+#ifdef _MSC_VER
```

Comment:
Defining `EIGEN_MAX_ALIGN_BYTES` maybe a solution.
Reference: https://eigen.tuxfamily.org/dox/TopicPreprocessorDirectives.html

Reply:
Thanks ! I'll read that...